lib/src/package_analysis/kind.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:analyzer/dart/element/element.dart';
+
+enum Kind { method, function, getter, setter }
+
+enum ParentKind { Class, Extension, Enum }
```

Comment:
OK to capitalise the values in this enum? ignore lint constant_identifier_names? `class`,  `extension` and `enum` are already keywords. maybe `classKind` (or `classK` for brevity) instead? ideally the notation for `Kind` and `ParentKind` should be identical.

Reply:
I would go with `classKind` etc.